fendermint/app/src/main.rs

```diff
@@ -3,30 +3,18 @@
 
 pub use fendermint_app_options as options;
 pub use fendermint_app_settings as settings;
-use ipc_observability::traces::{
-    register_tracing_subscriber, ConsoleLayerOpts, FileLayerOpts, WorkerGuard,
-};
+use ipc_observability::traces::{register_tracing_subscriber, WorkerGuard};
 
 mod cmd;
 
-fn init_tracing(_opts: &options::Options) -> Option<WorkerGuard> {
-    // let console_filter = opts.log_console_filter().expect("invalid filter");
-    // let file_filter = opts.log_file_filter().expect("invalid filter");
+fn init_tracing(opts: &options::Options) -> Option<WorkerGuard> {
```

Comment:
why not throw it out in `anyhow` and expect in `main`?

Reply:
Oh, because here we don't really care for an error. But rather the file guard might or might not be there (based on config) an either way is fine.